str/associatr/get_cis_numpy_files.py

```diff
@@ -207,6 +262,9 @@ def manage_concurrency_for_job(job: hb.batch.job.Job):
                 get_config()['get_cis_numpy']['remove_samples_file'],
                 snp_input,
                 get_config()['get_cis_numpy']['snp_loci'],
+                str_input,
+                get_config()['get_cis_numpy']['str_loci'],
+                get_config()['get_cis_numpy']['str_motifs'],
```

Comment:
you never need to pass values obtained using `get_config`, as the config is parsed into a global variable (you can just call ` get_config()['get_cis_numpy']['str_loci']` at the point you need the value)